src/map-interactions.react.js

```diff
@@ -45,15 +45,24 @@ function getMousePosition(el, event) {
 function getTouchPositions(el, event) {
   const points = [];
   const rect = el.getBoundingClientRect();
-  for (let i = 0; i < event.touches.length; i++) {
+  const touches = getTouches(event);
+  for (let i = 0; i < touches.length; i++) {
     points.push([
-      event.touches[i].clientX - rect.left - el.clientLeft,
-      event.touches[i].clientY - rect.top - el.clientTop
+      touches[i].clientX - rect.left - el.clientLeft,
+      touches[i].clientY - rect.top - el.clientTop
     ]);
   }
   return points;
 }
 
+function getTouches(event) {
```

Comment:
Would be good with a one or two line comment describing this function and why this is needed. Essentially, a one line version of your comment in the issue (#139).